kitsune/sumo/tests/test_email_utils.py

```diff
@@ -156,3 +156,20 @@ def test_send_messages(self, mock_mail):
         self.assertEqual(messages[1].to, ["georgeharrison@gmail.com"])
         self.assertEqual(messages[2].to, ["paulmccartney@gmail.com"])
         self.assertEqual(messages[3].to, ["ringo@beatles.com", "george@beatles.com"])
+
+    @patch("kitsune.sumo.email_utils.mail")
+    def test_send_messages_with_invalid_email(self, mock_mail):
+        from_email = "notifications@support.mozilla.org"
+        messages = [
+            EmailMultiAlternatives(
+                "Test",
+                "Testing",
+                from_email,
+                ["valid@example.com", "invalid@email", "another.valid@example.com"],
+            ),
+        ]
+        send_messages(messages)
+        send_messages_mock = mock_mail.get_connection().__enter__().send_messages
+        send_messages_mock.assert_called_once_with(messages)
+        # Check that the invalid email was removed
+        self.assertEqual(messages[0].to, ["valid@example.com", "another.valid@example.com"])
```

Comment:
`another.valid@example.com` is a valid email address. `invalid.@example.com` is not. The need for the gmail normalization is because for gmail addresses with dots (.) are the same. This is not true for other systems

Reply:
FWIW, the original test above did test an invalid email address ("ringo").